packages/docs/pages/popover.tsx

```diff
@@ -39,30 +46,27 @@ const PopoverPage = () => {
             }}
             {/* jsx-to-string:end */}
           </CodePreview>
-        </Panel>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => (
+        </>
+      </Panel>
+
+      <Panel header="Props" headerId="props">
         <PopoverPropTable
           inheritedProps={
             <>
               <BoxPropTable collapsible />
               <PaddingPropTable collapsible />
             </>
           }
+          renderPanel={false}
         />
-      ),
-    },
-  ];
+      </Panel>
 
-  return (
-    <>
-      <H1>Popover</H1>
-
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={['Tie popup to click or hover events on elements.', 'Use short texts.']}
+          discouraged={['Always display popup.', 'Extended texts that disrupt the screen.']}
```

Comment:
What does `'Use short texts.'` and `'Extended texts that disrupt the screen.'` mean? Very confused by this 🤔

Reply:
I'm trying to represent that popups aren't supposed to be used for longer texts/paragraphs. Not sure how to word it correctly, or if it's relevant.

Reply:
Hmm... maybe like `Use concise textual phrases within.`? Not sure how to rephrase the last one though 🤔

Reply:
I'm just gonna remove it.